BoxContentSDK/BoxContentSDK/Requests/BOXFileRequest.h

```diff
@@ -10,6 +10,11 @@
 @property (nonatomic, readwrite, assign) BOOL requestAllFileFields;
 @property (nonatomic, readwrite, strong) NSArray *notMatchingEtags;//If-None-Match: Array of strings representing etag values
 
+/*
+ * Set this string to "[hls]" request a representation endpoint for HLS video streaming.
```

Comment:
I'm also not sure about how to write an API that exposes this well. But we probably don't have time to make an awesome API with this change, so I suppose I'm ok with it for now if we can revisit later.